package.json

```diff
@@ -95,9 +96,11 @@
     "prettier": "2.0.5",
     "sinon": "9.0.2",
     "ts-node": "8.10.1",
+    "typemoq": "2.1.0",
```

Comment:
Todo: I don't think we use it

Reply:
Commited to v-ruurme/fix-mc-fus-feedback